platform_plugin_turnitin/handlers.py

```diff
@@ -1,12 +1,35 @@
 """Event handlers for the Turnitin plugin."""
 
+from django.conf import settings
+from opaque_keys.edx.keys import UsageKey
+
+from platform_plugin_turnitin.edxapp_wrapper.modulestore import modulestore
 from platform_plugin_turnitin.tasks import ora_submission_created_task
 
 
 def ora_submission_created(submission, **kwargs):
     """
     Handle the ORA_SUBMISSION_CREATED event.
 
+    Args:
+        submission (ORASubmissionData): The ORA submission data.
+    """
+    if settings.ENABLE_TURNITIN_SUBMISSION:
+        call_ora_submission_created_task(submission)
+        return
+
+    course_key = UsageKey.from_string(submission.location).course_key
+    course_block = modulestore().get_course(course_key)
+    enable_in_course = course_block.other_course_settings.get("ENABLE_TURNITIN_SUBMISSION", False)
```

Comment:
Actually, enabled_in_course could be a utility function. What do you think?

Reply:
So this looks similar to the filter definition.

Reply:
I think using a utility function is the best way to go.

Done: https://github.com/eduNEXT/platform-plugin-turnitin/commit/76af28f9f3aad18d2281696f9ce0164696a3482a